src/dom/dom.js

```diff
@@ -2122,29 +2130,6 @@ p5.Element.prototype.size = function(w, h) {
       } else if (aH === AUTO) {
         aH = w * this.height / this.width;
       }
-      // set diff for cnv vs normal div
-      if (this.elt instanceof HTMLCanvasElement) {
-        const j = {};
-        const k = this.elt.getContext('2d');
-        let prop;
-        for (prop in k) {
-          j[prop] = k[prop];
-        }
-        this.elt.setAttribute('width', aW * this._pInst._pixelDensity);
-        this.elt.setAttribute('height', aH * this._pInst._pixelDensity);
-        this.elt.style.width = aW + 'px';
-        this.elt.style.height = aH + 'px';
-        this._pInst.scale(this._pInst._pixelDensity, this._pInst._pixelDensity);
-        for (prop in j) {
-          this.elt.getContext('2d')[prop] = j[prop];
-        }
-      } else {
-        this.elt.style.width = aW + 'px';
```

Comment:
Looks like you lost the else part of the conditional here.

Reply:
Made both the changes @stalgiag